Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -98,7 +98,12 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
        .. code-block:: shell-session
 
           minikube start --network-plugin=cni
-          minikube ssh -- sudo mount bpffs -t bpf /sys/fs/bpf
+
+.. note::
+
+   From minikube v1.12.1+, cilium networking plugin can be enabled directly with
+   ``--cni=cilium`` parameter in ``minikube start`` command. However, this may not
+   install the latest version of cilium.
```

Comment:
I don't think we should have it, it's misleading and users might think it's an easier path to install Cilium, which it is, but installs an older Cilium version (1.8)

Reply:
I've already added a note about the version getting installed. It's useful to have a simpler step for users who don't care about version, and/or might miss the manual step.
Also, check this note from minikube command -

❗ With --network-plugin=cni, you will need to provide your own CNI. See --cni flag as a user-friendly alternative

Reply:
Given this is just a note to basically acknowledge that there is another path (but suggest why we don't advise that path by default), I'm OK with this. However I think it should be hidden inside the minikube group-tab, not at the outermost level of indentation.

Reply:
@joestringer Good catch! Fixed the formatting, PTAL. Thanks!